assets/scripts/streets/Geolocation.jsx

```diff
@@ -0,0 +1,156 @@
+/* global fetch */
+import React from 'react'
+import { connect } from 'react-redux'
+import { setSettings } from '../users/settings'
+import PropTypes from 'prop-types'
+import { Map, TileLayer, Marker, Popup } from 'react-leaflet'
+import SearchAddress from './SearchAddress'
+import {apiurlReverse, apikey} from './config'
+
+const OpenStreetMapTiles = 'http://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png'
```

Comment:
I'm curious of the Carto Basemaps might fit our color scheme better...? https://carto.com/location-data-services/basemaps/